### ISSUE TITLE
Minecraft to Discord chat input not outputting proper input. with messages like <`<5¢+µ³³:£9@«¶9¬%±>«2³²3$³³

### ISSUE THREAD

**Issue by FuzzyL16** [hypherionmc/sdlink#49]
Whenever I send a message from Minecraft chat it sends out "D3athboy22: <`<5¢+µ³³:£9@«¶9¬%±>«2³²3$³³"
I'm not sure if I've missed anything in Config 

I have also tried "Convert Discord to MC, and MC to Discord Formatting" to true or false but without success.

**Comment by hypherionmc** [comment]
Do you have any other mods that makes changes to chat? That almost looks like the message is encrypted somehow

**Comment by FuzzyL16** [comment]
No there are no mods just this one, I do think that Minecraft has added some kind of Encryption to chat for Chat Reports, and I believe they have added some kind of changes in 1.19.4. Although I have tried No Chat Reports mod to see if it would counteract some kind of encryption but with no success.

**Comment by hypherionmc** [comment]
I am aware of the encryption yeah, but I use this mod myself on both fabric and forge 1.19.4 and it's working fine on both servers, which is why it's odd. Is there anything different about your server, that wouldn't be part of a normal server?

**Comment by FuzzyL16** [comment]
I'm running this server on Aternos, could that be a problem? Although its still works with all other messages

**Comment by FuzzyL16** [comment]
I Have also just tried Minehut with the same problem, with the chat, I do know they are basically the same, I don't have a PC/setup to run on my own server to test.

I Can provide my Config if that can help?

**Comment by hypherionmc** [comment]
Aternos and minehut servers have a history of issue with this mod. Something about their platform causes the mod to misbehave in weird ways from not even starting the bot, all the way to crashing on startup without any clear errors. I don't 100% believe that is the issue in this case though.

Can you maybe send me a list of all the mods you have on the server?

**Comment by FuzzyL16** [comment]
I am running fabric 1.19.4 (0.14.18) with mods SimpleDiscordLink-fabric-1.19.3-2.2.1.jar and of course 
fabric-api-0.77.0+1.19.4.jar

This was just a test server before I would put it on my main server since it wasn't working properly, from bot perms, got that fixed but the chat had this problem.

**Comment by hypherionmc** [comment]
Interesting. Well thanks for the info. Let me see if I can get a server from one of the two to test this myself. Really curious as to why their platforms are always a problem with this mod. I have contacted their support as well about it, but they just never got back to me.

Will let you know on my findings

**Comment by hypherionmc** [comment]
FINALLY got an open slot from Aternos.

Mod is working as expected with both webhooks, normal embeds and normal text messages. Only difference is I used Fabric 0.14.19 (as that's what they had)

![image](https://user-images.githubusercontent.com/63595580/232905744-d4e3aeab-4cc5-4197-b6ea-0d83fcebfa82.png)

**Comment by FuzzyL16** [comment]
So did I just have an outdated Fabric API?

**Comment by FuzzyL16** [comment]
Ok I've did more digging and found what was the cause.

I'm not sure how the game gives the mod the chat messages, But this bug is due to client slide mods. Any Unsigned messages WILL come out encrypted, since I use No Chat Reports on Client it doesn't send the raw message, I tested this with a Vanilla Client and turned the setting off with the mod and it works Perfectly.

I'm not sure on how you work with mod compatibility, but I and lots of others use No Chat Reports. I hope to see this fixed because this mod works great and I would love to add it to my main Server.
